src/main/java/org/takes/http/BkBasic.java

```diff
@@ -70,17 +77,34 @@ public BkBasic(final Take tks) {
         this.take = tks;
     }
 
+//      @todo #519:30min Need to support Keep-Alive header here and take
```

Comment:
@exper0 you have to big indentation inside the comment, `//` should be at the same level as the method, and the comment should start after a single space (todo continuation should start at two spaces)